src/v2/Components/NavBar/useNavBarHeigth.ts

```diff
@@ -0,0 +1,11 @@
+import { themeProps } from "@artsy/palette"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+import { MOBILE_NAV_HEIGHT, NAV_BAR_HEIGHT } from "./constants"
+
+export const useNavBarHeigth = (
+  mobileMediaQueryString: string = themeProps.mediaQueries.xs
+) => {
+  const isMobile = useMatchMedia(mobileMediaQueryString)
```

Comment:
We'll want to be sure that this works during the server-side render pass and with JS turned off

Reply:
@damassi is there a way how we identify a user's device at the server-side render time?

`useMatchMedia` doesn't work during the server-side render
https://github.com/artsy/force/blob/89d35f7144aafe4987967f9ac774783bfa2c2374/src/v2/Utils/Hooks/useMatchMedia.ts#L40-L43

Reply:
I'm not entirely sure about the context around this bit of code, but the way we genenerally do server-side compatable layouts is via our library fresnel: https://github.com/artsy/fresnel. 

There's quite a few examples of its use throughtout our codebase; just search for `<Media at` or `<Media greaterThan=`. With those pieces one can do some clever things like:

```
<Media at='sm'>{renderLayout({ isMobilie: true })}</Media>
```

Not the prettiest, but it works for when one needs an `isMobile`-ish variable.

Reply:
I rewrote the components to use `Media` component, please take a look